R/OneClassTask.R

```diff
@@ -0,0 +1,30 @@
+#' @export
+#' @rdname Task
+
+makeOneClassTask = function(id = deparse(substitute(data)), data,
+  weights = NULL, blocking = NULL, fixup.data = "warn", check.data = TRUE) {
+  assertString(id)
+  assertDataFrame(data)
+  # some code on cran passed stuff like positive=1, we can live with the convert here
+  # for one class positive is always set to the one class
+  assertChoice(fixup.data, choices = c("no", "quiet", "warn"))
+  assertFlag(check.data)
+
+  task = makeUnsupervisedTask("oneclass", data, weights, blocking,
+    fixup.data = fixup.data, check.data = check.data)
+
+  task$task.desc = makeOneClassTaskDesc(id, data, weights, blocking)
+  addClasses(task, "OneClassTask")
+
+}
+
+makeOneClassTaskDesc = function(id, data, weights, blocking) {
+  target = character(0L)
+  td = makeTaskDescInternal("oneclass", id, data, target, weights, blocking)
+  return(addClasses(td, c("OneClassTaskDesc", "UnsupervisedTaskDesc")))
+}
+
+#' @export
+print.OneClassTask = function(x, ...) {
+  print.UnsupervisedTask(x)
```

Comment:
Thanks for the review :) 
I will add the ellipsis, and I guess Classiftask.R and Clustertask.R (and probably all other tasks) also need to add ellipsis, when calling `print.Supervised()`/`print.Unsupervised()` in their print-fct.

Can you explain what you mean by `NextMethod()`?